benchexec/tools/sbt-fizzer.py

```diff
@@ -0,0 +1,148 @@
+import benchexec.tools.template
+import benchexec.result as result
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import os
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for sbt-fizzer.
+    """
+
+    def name(self):
+        """
+        Return the name of the tool, formatted for humans.
+        This method always needs to be overriden, and typically just contains
+        return "My Toolname"
+        @return a non-empty string
+        """
+        return "sbt-fizzer"
+
+    def executable(self, tool_locator):
+        """
+        Find the path to the executable file that will get executed.
+        This method always needs to be overridden,
+        and should typically delegate to our utility method find_executable. Example:
+        return tool_locator.find_executable("mytool")
+        The path returned should be relative to the current directory.
+        @param tool_locator: an instance of class ToolLocator
+        @return a string pointing to an executable file
+        """
+        return tool_locator.find_executable("sbt-fizzer.py")
+
+    def version(self, executable):
+        """
+        Determine a version string for this tool, if available.
+        Do not hard-code a version in this function, either extract the version
+        from the tool or do not return a version at all.
+        There is a helper function `self._version_from_tool`
+        that should work with most tools, you only need to extract the version number
+        from the returned tool output.
+        @return a (possibly empty) string
+        """
+        return ""
+
+    def program_files(self, executable):
+        """
+        OPTIONAL, this method is only necessary for situations when the benchmark environment
+        needs to know all files belonging to a tool
+        (to transport them to a cloud service, for example).
+        Returns a list of files or directories that are necessary to run the tool,
+        relative to the current directory.
+        The default implementation returns a list with the executable itself
+        and all paths that result from expanding patterns in self.REQUIRED_PATHS,
+        interpreting the latter as relative to the directory of the executable.
+        @return a list of paths as strings
+        """
+        return [executable] + self._program_files_from_executable(executable, ["lib", "lib32", "tools"], parent_dir=False)
```

Comment:
Please overwrite only those methods that you actually need to overwrite. Instead of duplicating the code from `program_files`, you can simply declared a fields `REQUIRED_PATHS = ["lib", "lib32", "tools"]`.